platforms/lambda/prisma/schema.prisma

```diff
@@ -1,6 +1,6 @@
 generator client {
   provider      = "prisma-client-js"
-  binaryTargets = ["native", "rhel-openssl-1.0.x"]
+  binaryTargets = ["native"]
```

Comment:
`"rhel-openssl-1.0.x"` is needed for Lambda

See https://www.prisma.io/docs/guides/deployment/deploying-to-aws-lambda#binary-targets-in-schemaprisma

Reply:
Native is to make it works on your local machine

Reply:
Why do we end up with `node_modules/@prisma/engines/query-engine-debian-openssl-1.1.x` then?

Reply:
so we end up with 

From .prisma/client (to use for @prisma/client)
  adding: node_modules/.prisma/client/query-engine-rhel-openssl-1.0.x (deflated 64%)
  adding: node_modules/.prisma/client/query-engine-debian-openssl-1.1.x 

From @prisma/engines (binaries for prisma cli only)
  adding: node_modules/@prisma/engines/introspection-engine-debian-openssl-1.1.x (deflated 65%)
  adding: node_modules/@prisma/engines/query-engine-debian-openssl-1.1.x (deflated 64%)
  adding: node_modules/@prisma/engines/prisma-fmt-debian-openssl-1.1.x (deflated 70%)
  adding: node_modules/@prisma/engines/migration-engine-debian-openssl-1.1.x (deflated 63%)

Reply:
Here the CI native is ubuntu which is Debian

Reply:
Ah, makes sense! Fixing!

Reply:
Just fixed in a comment 😄

Reply:
Oh hah, woops. My github workflows are a bit dated 😅